index.js

```diff
@@ -3,19 +3,277 @@ const links= document.querySelector('#navlinks');
+ language: ['Html', 'Bootstrap', 'Ruby'],
+ button: "See Project"
+},
+
+{
+ heading: "Professional Art ",
+ subHeading: "Printing Data",
+ paragragh: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+ language: ['Html', 'Bootstrap', 'Ruby'],
+ button: "See Project"
+},
+];
+pens.forEach((pen) => {
+ const first = `
+ <div class="cart" id="cf1">
+ <h2>${pen.heading}<span>${pen.subHeading}</span></h2>
+ <p>${pen.paragragh}</p>
+ <ul>
+ <li>${pen.language[0]}</li>
+ <li>${pen.language[1]}</li>
+ <li>${pen.language[2]}</li>
+ </ul>
+ <button id="press" type="button" class="click" name="See-Project">${pen.button}</button>
+ </div>`;
+ carttt.innerHTML += first;
+});
+
+//start of desktop carts
+const cartt2 = document.querySelector('.cartt2');
+const books = [{
+ image: "Resources/Images/Mask-Group.png",
+ alt: "printing pic",
+ button: "See-Project"
+},
+
+{
+ image: "Resources/Images/ImgPlaceholder(9).png",
+ alt:"printing pic", //single background and card
+ heading: "Data Dashboard <br> Healthcare",
+ paragraph: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+ language: ['Html', 'Bootstrap', 'Ruby'],
+ button: "See Project"
+},
+
+{
+ image: "Resources/Images/Mask-Group2.png",
+ alt: "art painting",
+ heading: "Website Protfolio",
+ paragraph: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+ language: ['Html', 'Bootstrap', 'Ruby'],
+ button: "See Project"
+},
+
+{
+ image: "Resources/Images/imageplace4.png",
+ alt:"image picture",
+ heading: "Profesional Art",
+ subHeading: "Printing Data More",
+ paragraph: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+ language: ['Html', 'Bootstrap', 'Ruby'],
+ button: "See Project"
+},
+
+{
+ image: "Resources/Images/ImgPlaceholder(9).png",
+ alt: "printing pic",
+ heading: "Data Dashboard Healthcare",
+ paragraph: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+ language: ['Html', 'Bootstrap', 'Ruby'],
+ button: "See Project"
+},
+
+{
+ image: "Resources/Images/Mask-Group2.png",
+ alt: "art painting",
+ heading: "Website Protfolio",
+ paragraph: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+ language: ['Html', 'Bootstrap', 'Ruby'],
+ button: "See Project"
+}
+];
+
+let cards = document.querySelector('.cartt2');
+cards.innerHTML = `<div class="cartbox" id="c1">
+<img class="proj1" src="${books[0].image}" alt="${books[0].alt}"/>
+
+ <button id="press" class="firstbut click" type="button" name="See-Project">${books[0].button}</button>
+</div>
+
+<div class="cart c2">
```

Comment:
- [OPTIONAL] I think it's better to create this part in a loop as done in the mobile version or use a unique section for both versions. As the unique section was not required by the previous reviewer I consider this optional. The use of the loop is a suggestion.